lib/index.js

```diff
@@ -143,22 +87,19 @@ function parse(opts, callback) {
     assert.string(opts.body, 'opts.body');
     assert.func(callback, 'callback');
 
-    const writeStream = through2.obj(function(chunk, enc, cb) {
-        this.push(chunk);
-        return cb();
-    });
-    const parseStream = createParseStream(opts);
+    const sourceStream = intoStream(opts.body);
```

Comment:
Conversion of in memory string to into readable chunks is going to help a lot - I suspect we were losing I/O here because pushing a giant string into a stream effectively becomes a giant JSON.parse